dataset-cellsystems-2021/dataset.json

```diff
@@ -1,6 +1,7 @@
 {
     "title": "Structural organization and gene expression in hiPSC-derived cardiomyocytes",
     "name": "cellsystems",
+    "dateCreated": "Jul 22, 2021",
```

Comment:
should there be separate dates in the live and fish sub-datasets?

Reply:
since this is used for just sorting the main cards it didn't seem needed. The datasets themselves have version numbers, so we could always sort by version within a megaset, but at the moment that is seen as unnecessary

Reply:
do the child datasets inherit the megaset creation date?

Reply:
it's not stored there, i can add it, but yeah implicitly, the date created is for the whole set of them.